generate_custom_slots.py

```diff
@@ -215,3 +215,42 @@
 for a in cleaned:
   gfile.write("%s\n" % a.encode("utf-8"))
 gfile.close()
+
+# Generetate CHANNELS Slot
+retrieved = kodi.GetPVRChannels()
+
+all = []
+
+if 'result' in retrieved and 'channels' in retrieved['result']:
+  for v in retrieved['result']['channels']:
+    ascii_name = v['sanitized_label'].encode('ascii', 'replace')
```

Comment:
Should this be ASCII-only?  We support UTF-8 in the other slots.

You also still need to call kodi.sanitize_name() (or a subset of it, if it doesn't all apply to channels) because there are characters that Amazon restricts in slot items.